Formula/pyoxidizer.rb

```diff
@@ -0,0 +1,22 @@
+class Pyoxidizer < Formula
+  desc "Modern Python application packaging and distribution tool"
+  homepage "https://github.com/indygreg/PyOxidizer"
+  url "https://github.com/indygreg/PyOxidizer/archive/pyoxidizer/0.17.tar.gz"
+  sha256 "9117d411b610e29dfd8d9250cd1021afb545550fd7e698b623e913c26114f013"
+  license "MPL-2.0"
+  head "https://github.com/indygreg/PyOxidizer.git", branch: "main"
+
+  depends_on "rust" => :build
+  depends_on xcode: :test if MacOS.version <= :catalina
```

Comment:
Shouldn't this be a runtime dependency too?

Reply:
I'm only experimenting as it looks like test block is failing due to upstream issue. Don't want to force full XCode dependency right now until upstream comments.

Reply:
Still failed on Mojave. Currently seems to only work with macOS 11 SDK, so minimum Xcode 12.2, which means minimum macOS of 10.15.4.

We could add this as a temporary runtime dependency and merge now, and remove if upstream fixes issue.

Reply:
Opened a new upstream issue to track this: https://github.com/indygreg/PyOxidizer/issues/431